cve_bin_tool/cvedb.py

```diff
@@ -321,7 +328,7 @@ def get_cvelist_if_stale(self) -> None:
     def latest_schema(
         self,
         table_name: str = "",
-        table_schema: str = "",
+        table_schema="",
```

Comment:
Is this actually not a string any more?  I don't see any other changes to table_schema so I'm not sure why this was changed.

Reply:
Here, we're using pop() function on a string, and _mypy_ was giving trouble. So made the temporary change.
https://github.com/intel/cve-bin-tool/blob/42e786faeb0f04ff2614a34c5a0cce442d669e4b/cve_bin_tool/cvedb.py#L351-L354